### PR TITLE
Update posts

### PR DESCRIPTION
**Pull request by A13xYanez**

### CODE REVIEW

File: src/controllers/posts.js [hunk 1]
```diff
@@ -99,3 +100,25 @@ export const deletePost = async (req, res) => {
     return res.status(500).json({ error: 'Internal server error' });
   }
 };
+
+// update posts
+export const updatePost = async (req, res) => {
```
Comment (JJeshua) [inline src/controllers/posts.js:96]: no need to get the post id from the req params, already checked from postExists middleware. Use lodash get. also no need to check the post id is valid, already checked in middleware.

File: src/controllers/posts.js [hunk 2]
```diff
@@ -99,3 +100,25 @@ export const deletePost = async (req, res) => {
...
+      postUpdate(post_id, {
+        post_content,
+      });
+      return res.status(200).json({ message: 'Post updated successfully' });
+    } else {
+      return res.status(404).json({ error: 'Post not found' });
+    }
+  } catch (error) {
+    console.error('Error updating post: ', error);
+    return res.sendStatus(400).json({
```
Comment (JJeshua) [inline src/controllers/posts.js]: should be 500 error, and res.status(500).json(...

**Review by JJeshua — APPROVED** [review]
LGTM.

**Review by vlomeli — APPROVED** [review]
"message": "Post updated successfully" and changed the post on mongo db. Works good good!

**Pull request merged**